charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All changes to this chart will be documented in this file.
 * Changed default value for caCerts.image
 * Document the upgrade process when autoscaling is enabled
 * Fix openshift change-admin-password-hook Job SecurityContext failure
+* Update helm chart repo path in sources for each Chart
```

Comment:
This message is different from the one in `Chart.yml`, we usually try to keep the same for consistency.

You can remove 'for each Chart' considering the Changelog is in a chart itself.

Reply:
Ho yes sorry about that. Fixed and rebased.